.byebug_history

```diff
@@ -0,0 +1,13 @@
+s
+display arg
+s
+display ind
+s
+display array
+s
+display &prc
+display prc
+display arg
+display ind
+display array
+s
```

Comment:
Files not related to the project should not be committed and added to .gitignore ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] Delete `.byebug_history` from the repo and add it to `.gitignore`.